### PR TITLE
Add TNT s model

### PR DESCRIPTION
**Pull request by Rishit-dagli**
This PR adds the Transformer iN Transformer (TNT) model that uses pixel level attention paired with patch level attention for image classification. The TNT model was proposed in the paper "Transformer in Transformer" by Han et al. for image classification pre-trained on ImageNet-1K. TNT S with (just) 23.8 M parameters and 5.2 GFLOPS achieves 81.4% top-1 accuracy and 95.7% top-5 accuracy on ImageNet-1K.

The original NeurIPS 2021 paper can be found [here](https://proceedings.neurips.cc//paper/2021/hash/854d9fca60b4bd07f9bb215d59ef5561-Abstract.html) 

---

Any pull request you open is subject to the TensorFlow Hub Terms of Service at www.tfhub.dev/terms and Google's Privacy Policy at https://www.google.com/policies/privacy.

We check modified Markdown files for validity using [this](https://github.com/tensorflow/tfhub.dev/blob/master/.github/workflows/contributions-validator.yml) GitHub Workflow. You can execute the same checks locally by passing the file paths of modified Markdown files (relative to the `assets/docs` directory) to validator.py e.g. `python3 ./tools/validator.py google/google.md google/models/albert_base/1.md ...`.

### CODE REVIEW

File: assets/docs/rishit-dagli/tnt-s/1.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    image_resized = tf.cast(image_resized, tf.float32)
+    image_resized = (image_resized - 127.5) / 127.5
+    image_resized = tf.keras.layers.Normalization(
+        mean=(0.5, 0.5, 0.5), variance=(0.25, 0.25, 0.25)
+    )(image_resized)
+    return tf.expand_dims(image_resized, 0).numpy()
+
+
+def load_image_from_url(url):
+    response = requests.get(url)
```
Comment (WGierke) [inline assets/docs/rishit-dagli/tnt-s/1.md:71]: I think you still need to import `requests` and `Image` (from `PIL` I guess?)

File: assets/docs/rishit-dagli/tnt-s/1.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+)
+infer_on_image(
+    img_url="https://i.imgur.com/Wv99De3.jpg", expected_label="window_screen"
+)
+```
+
+### Notes
+
+- The original model weights are provided as a PyTorch model [2]. They were ported to a TensorFlow SavedModel.
+- The model inputs should be four dimensional Tensors of the shape (batch_size, height, width, num_channels)` in the range `[-1, 1]`.
```
Comment (WGierke) [inline assets/docs/rishit-dagli/tnt-s/1.md:103]: Nit: please add a `

**Review by WGierke — CHANGES_REQUESTED** [review]
Thank you, @Rishit-dagli ! The documentation looks great! I only left two minor comments

File: assets/docs/rishit-dagli/tnt-s/1.md (hunk 1 above)
Comment (Rishit-dagli) [reply assets/docs/rishit-dagli/tnt-s/1.md:71]: Ah, yes, added that

**Review by WGierke — APPROVED** [review]
Great, thanks a lot!
